packages/discovery-react-components/src/components/CIDocument/components/Section/Section.tsx

```diff
@@ -1,3 +1,6 @@
+/* eslint-disable jsx-a11y/no-static-element-interactions */
```

Comment:
I have some guesses as to what this component does, but currently our cluster is down and it's preventing me from being confident in the exact functionality. This rule, and the one below it, are the two `a11y` violations in this repo

`click-events-have-key-events` states that anything you can click, you should also be able to navigate with the keyboard and select. the only problem is that the onClick handler `mouseClickListener` takes a `mouseEvent` and not a `keyboardEvent`. I'm not able to say that this function can take a `mouseEvent` _or_ `keyboardEvent` because this handler makes use or mouse positioning `client.X` and `client.Y` and I'm unsure of just using the current cursor position is ideal in this scenario and would love some feedback as to what might be the best approach here (if not disabling the rule)

`no-static-element-interactions` states that you need to provide a `role` to this element. the list of options can be found [here](https://github.com/evcohen/eslint-plugin-jsx-a11y/blob/master/docs/rules/no-static-element-interactions.md#case-this-element-acts-like-a-button-link-menuitem-etc). I originally set the `role` to `button`, but in that case you need to make choices about what the tab-ability should be, in this case some feedback would also be helpful (if not to disable this rule too)

Reply:
after adding in `eslint-config-react-app`, these accessibility rules no longer apply